### PR TITLE
Elasticsearch Logging Middleware

### PR DESCRIPTION
**Pull request by adityaprasanna**
- Created 2 APIs to interact with elasticsearch
- Stores logs in elasticsearch_logs_db
- Acts as a middleware and all requests must go through this service

### CODE REVIEW

File: src/logs/elasticsearch_logs/api/views.py [hunk 1]
```diff
@@ -22,8 +22,68 @@ class Query(APIView):
...
+                    "client_id":data["client_id"],
+                    "room_id":data["room_id"],
+                    "message_id":data["message_id"],
+                    "query_id":"null",
+                    "query":data["question"],
+                    "question_matched":"null",
+                    "answer":"null",
+                    "action":"null",
+                    "action_param":"null",
+                    "score":0,
```
Comment (avinassh) [inline src/logs/elasticsearch_logs/api/views.py]: this isn't formatted nicely, can you do it? like spaces after `:` etc

File: src/logs/elasticsearch_logs/api/views.py [hunk 2]
```diff
@@ -22,8 +22,68 @@ class Query(APIView):
...
+                "client_id":data["client_id"],
+                "room_id":data["room_id"],
+                "message_id":data["message_id"],
+                "query_id":response_data["_id"],
+                "query":data["question"],
+                "question_matched":response_data["_source"]["Question"],
+                "answer":response_data["_source"]["Answer"],
+                "action":response_data["_source"]["Action"],
+                "action_param":response_data["_source"]["Action-Param"],
+                "score":response_data["_score"],
```
Comment (avinassh) [inline src/logs/elasticsearch_logs/api/views.py]: same here

Comment (adityaprasanna) [reply src/logs/elasticsearch_logs/api/views.py]: done.

File: src/logs/elasticsearch_logs/api/views.py (hunk 1 above)
Comment (adityaprasanna) [reply src/logs/elasticsearch_logs/api/views.py]: done.

**Pull request merged**